src/storage/qml.js

```diff
@@ -2,8 +2,58 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-module.exports = {
-    getItem(key) { },
+const DB_NAME = "Glean";
+
+// Initialize the database ASAP.
```

Comment:
Okay, I had initially wrapped this all up in a `QMLStorage` class, but for some reason that failed in QML. It complained about `this` being undefined unless inside the constructor. There is something minor going on there, since we organize the code in this repo with classes all over, and QML doesn't complain for other modules. I couldn't figure out what was the issue so I leave it like this for now.

Reply:
Ok, but please add a comment here explaining this, so that we don't forget while creating the real GleanJS.

Reply:
Sure, I expect that this issue will just not happen in the real GleanJS. But I'll add a comment just to be sure.